Marlin/src/inc/Conditionals_post.h

```diff
@@ -469,6 +473,45 @@
   #undef HEATER_1_MAXTEMP
 #endif
 
+#if MAX6675_0_IS_MAX31855 || MAX6675_1_IS_MAX31855
+  #define MAX6675_HAS_MAX31855 1
+  #if PIN_EXISTS(MAX6675_DO) || PIN_EXISTS(MAX31855_MISO)
+    #define MAX31855_USES_DO 1
+  #endif
+  #if PIN_EXISTS(MAX6675_SCK) || PIN_EXISTS(MAX31855_SCK)
```

Comment:
It's weird for me. How can a SPI device work with a clock PIN?

Reply:
The SCK or clock PIN is only used when you are bit banging for Software SPI